sf_gov_departments.csv

```diff
@@ -1,53 +1,56 @@
-"Human Resources, Director","Department of Human Resources","Campaign & Governmental Conduct Code Sec. 3.1-103(b)(1)"
-"Human Rights Commission, Executive Director","Human Rights Commission","Campaign & Governmental Conduct Code Sec. 3.1-103(b)(1)"
-"Human Services Agency, Executive Director","Human Services Agency","Campaign & Governmental Conduct Code Sec. 3.1-103(b)(1)"
-"Juvenile Probation Commission, Chief Probation Officer","Juvenile Probation Department","Campaign & Governmental Conduct Code Sec. 3.1-103(b)(1)"
-"Mayor","Office of the Mayor","Campaign & Governmental Conduct Code Sec. 3.1-103(b)(1)"
-"Municipal Transportation Agency, Executive Director/CEO","Municipal Transportation Agency","Campaign & Governmental Conduct Code Sec. 3.1-103(b)(1)"
-"Planning, Director","Planning Department","Campaign & Governmental Conduct Code Sec. 3.1-103(b)(1)"
-"Police Chief","Police Department","Campaign & Governmental Conduct Code Sec. 3.1-103(b)(1)"
-"Port, Director","Port","Campaign & Governmental Conduct Code Sec. 3.1-103(b)(1)"
-"Public Defender","Public Defender's Office","Campaign & Governmental Conduct Code Sec. 3.1-103(b)(1)"
-"Public Health, Director","Department of Public Health","Campaign & Governmental Conduct Code Sec. 3.1-103(b)(1)"
-"Public Works, Director","Department of Public Works","Campaign & Governmental Conduct Code Sec. 3.1-103(b)(1)"
-"Recreation and Park, General Manager","Recreation and Park Department","Campaign & Governmental Conduct Code Sec. 3.1-103(b)(1)"
-"Residential Rent Stabilization and Arbitration Board, Executive Director","Rent Board","Campaign & Governmental Conduct Code Sec. 3.1-103(b)(1)"
-"Retirement System, Executive Director","Retirement System","Campaign & Governmental Conduct Code Sec. 3.1-103(b)(1)"
-"San Francisco Public Utilities Commission, General Manager","Public Utilities Commission","Campaign & Governmental Conduct Code Sec. 3.1-103(b)(1)"
-"Sheriff","Sheriff's Department","Campaign & Governmental Conduct Code Sec. 3.1-103(b)(1)"
-"Sheriff's Department Office of Inspector General, Inspector General","Sheriff's Department Office of Inspector General","Campaign & Governmental Conduct Code Sec. 3.1-103(b)(1)"
-"Small Business, Director","Office of Small Business","Campaign & Governmental Conduct Code Sec. 3.1-103(b)(1)"
-"Technology, Executive Director","Department of Technology","Campaign & Governmental Conduct Code Sec. 3.1-103(b)(1)"
-"Treasurer","Office of the Treasurer & Tax Collector","Campaign & Governmental Conduct Code Sec. 3.1-103(b)(1)"
-"War Memorial and Performing Arts Center, Managing Director","War Memorial","Campaign & Governmental Conduct Code Sec. 3.1-103(b)(1)"
\ No newline at end of file
+﻿"Department Heads","Departments","Departments, Legal Source","2024-2025 Full-Time Employees (FTE)","2025-2026 Full-Time Employees (FTE)"
+"Academy of Sciences, Executive Director","California Academy of Sciences","Charter Sec. 16.105","13.34","13.34"
```

Comment:
Their org chart from https://www.sf.gov/sites/default/files/2024-05/CSF_Proposed_Budget_Book_June_2024_r8.pdf has Board of Trustees at the top, but I thought Executive Director made more sense...

<img width="1203" alt="Screenshot 2024-10-18 at 5 42 37 PM" src="https://github.com/user-attachments/assets/fa8c2ab2-fade-4c6a-8155-c6e4e3d2234d">

Reply:
Yeah Board of Trustees would be "commission" level and Executive Director would be department head level